vulture_allowlist.py

```diff
@@ -290,6 +290,11 @@
 extra
 allow_mutation
 validate_assignment
+linux_filename_valid
+windows_filename_valid
+blocked_ips_valid
+inaccessible_subnets_valid
+subnets_valid
```

Comment:
import these or import ScanTargetConfiguration and do `ScanTargetConfiguration.subnets_valid`